datasets/caldor-fire-characteristics-burn-severity.data.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: caldor-fire-behavior-burn-severity
+name: "Caldor Fire Behavior and Burn Severity"
+description: "Assets describing the progression and active fire behavior of the 2021 Caldor Fire in California."
+media:
+  src: ::file ./caldor-fire--dataset-cover.jpg
+  alt: Hillside engulfed by a wildfire.
+  author:
+    name: Marek Piwnicki
+    url: https://unsplash.com/photos/WiZOyYqzUss
+thematics:
+  - eis
+layers:
+  - id: caldor-fire-behavior
+    stacCol: caldor-fire-behavior
+    name: Fire Behavior
+    type: raster
+    description: ""
+    initialDatetime: newest
+    zoomExtent:
+      - 10
+      - 20
+    sourceParams:
+      colormap_name: inferno_r
+      rescale:
+        - -41
+        - 98
```

Comment:
The new files apparently have a different min max value? (ref: the image in this PR conversation).
I reran the ingestion pipeline - so the min max value have changed:
check here: https://staging-stac.delta-backend.com/collections/caldor-fire-behavior/items